packages/terra-clinical-slide-group/docs/README.md

```diff
@@ -0,0 +1,27 @@
+# Terra Clinical Slide Group
+
+The SlideGroup is a Component that utilizes the `react-transition-group` library to present a stack of components in an
+animated fashion.
+
+The last component in the group is visible, and all others are hidden. While the hidden components are not visible and
+marked with the `aria-hidden` attribute, they are not actually unmounted.
```

Comment:
do we anticipate this is going to cause problems with people throwing id's on stuff?